docs-web/configuration/policy-resource.md

```diff
@@ -18,6 +18,8 @@ This document is the reference documentation for the Policy resource. An example
       - [AccessControl Merging Behavior](#accesscontrol-merging-behavior)
     - [RateLimit](#ratelimit)
       - [RateLimit Merging Behavior](#ratelimit-merging-behavior)
+    - [JWT](#jwt)
+      - [JWT Merging Behavior](#jwt-merging-behavior)
```

Comment:
the main table also needs to be updated. the `jwt` field needs to be documented.